lib/core/lib/ops/policyOps.js

```diff
@@ -310,6 +310,12 @@ const policyOps = {
           organization_policies
         WHERE
           org_id = ${organizationId}
+      ),
```

Comment:
Your queries fix the bug because you implement what you think it happens, but i think the problem here is that the policies are not "scoped per user".
1) User polices are clearly scoped because you set it in the query directly
2) Team policies for the user are again scoped because first you find all the teams PER USER and than just take the policies
3) The problem is with organization because the userId is not involved in the query
I'd go for something like this:
```
SELECT policy_id
FROM organization_policies op
LEFT JOIN users u ON u.org_id = op.org_id
WHERE u.id = ${userId}
```
WDYT?

Reply:
You understood it well, this was the issue. There are two problems: org policies not scoped by user and a smaller one that user id not checked. that it exists.

Reply:
i also considered this type of query but this doesn't take into account the impersonation that complicates a little the approach. Good suggestion, will rethink how the query can be improved.